### PR TITLE
[ASGenerator] Assign statement to variable for '1 + 2'. fixes #1749

### LINKED ISSUE #1749: [Bug][ASGenerator] Wrong result after execution `Assign statement to variable` for `1 + 2;`
```haxe
1 + 2; | //  | <-- cursor
```
actual result:
```haxe
1 + var float:Float = 2;
```
expected result:
```haxe
var float:Float = 1 + 2;
```

### PR DESCRIPTION
**Pull request by SlavaRa**

### CODE REVIEW

File: External/Plugins/AS3Context/Context.cs [hunk 1]
```diff
@@ -131,7 +131,7 @@ public Context(AS3Settings initSettings)
             features.privateKey = "private";
             features.intrinsicKey = "extern";
             features.namespaceKey = "namespace";
-
+            features.ArithmeticOperators = new HashSet<char>{'+', '-', '*', '/'};
```
Comment (Neverbirth) [inline External/Plugins/AS3Context/Context.cs]: What about %, |, ^, etc?

Comment (SlavaRa) [reply External/Plugins/AS3Context/Context.cs]: I will add all this after this requester, but gradually

File: External/Plugins/ASCompletion/Completion/ASComplete.cs [hunk 2]
```diff
@@ -5144,6 +5147,7 @@ sealed public class ASExpr
         public string Value;
         public List<string> SubExpressions;
         public char Separator;
+        public int SepparatorPosition;
```
Comment (Neverbirth) [inline External/Plugins/ASCompletion/Completion/ASComplete.cs]: Spelling is wrong.

**Comment by Neverbirth** [comment]
Conflicts now.

File: External/Plugins/ASCompletion/Completion/ASComplete.cs (hunk 2 above)
Comment (SlavaRa) [reply External/Plugins/ASCompletion/Completion/ASComplete.cs]: Sorry. Moment :)

**Comment by SlavaRa** [comment]
5 min.

**Pull request merged**